docs/deployment/server-configuration.md

```diff
@@ -224,3 +224,10 @@ While serving multiple databases over the same API with pREST is doable, it's by
 **Cross-Origin Resource Sharing**
 
 Read the specific topic where we talk about CROS [here](/prestd/deployment/cors-support/).
+
+
+## Health check endpoint
+
+If you need to setup a health check on your deployment (ECS/EKS or others), you can use `/_health` as a provider of this information.
+
+The server will return 503 whenever a Postgres connection is not reachable.  
```

Comment:
The general use of a health check is to know that the web server is responding (not that the database is reachable). The database being reachable is a requirement of the web server.

Reply:
Should the description be: `The server will return 503 whenever the web server and pREST requirements are not responding, such as Postgres connection or querying access`?